tests/loaddata/test_multiomics_datasets.py

```diff
@@ -0,0 +1,177 @@
+import os
+
+import torch
+from torch_geometric.data import Data, DataLoader
+
+import kale.prepdata.tabular_transform as T
+from kale.loaddata.multiomics_datasets import MultiOmicsDataset, SparseMultiOmicsDataset
+from kale.utils.seed import set_seed
+
+
+def test_multiomics_datasets():
```

Comment:
Is this necessary? Will this be tested anyway by the following tests?

Reply:
The MultiOmicsDataset class inherits from the MessagePassing class in PyTorch Geometric, requiring complete testing of all properties (such as x, y, train_idx, test_idx, etc) to ensure proper settings.